### PR TITLE
Adds image upload via Cordova plugin

### PR DESCRIPTION
**Pull request by tonytlwu**
Image uploads are defined with `<input type="file" data-file-image name="fieldName">` to make use of [Cordova's Camera plugin](https://github.com/apache/cordova-plugin-camera), which takes care of stripping EXIF data and resizing. Resizing is taken care of on the web through an off-page `<canvas>`.

Images will be resized to contain within 1024x1024 by default and are compressed as 80% quality JPEG. Additional `data-width` and `data-height` can be added to customise the resizing options.

Images uploaded can be previewed through a `<canvas>` element with `data-file-name="fieldName"` to match the file input.

### CODE REVIEW

File: js/build.js [hunk 1]
```diff
@@ -99,9 +103,66 @@
...
+      var files = e.target.files;
+      selectedFileInputName = e.target.name;
+      var customWidth = $(e.target).attr('data-width') || 1024;
+      var customHeight = $(e.target).attr('data-height') || 1024;
+      var file;
+      for (var i = 0, l = files.length; i < l; i++) {
+        if (i > 0) return; // Restrict support to only 1 file at the moment
+
+        file = files[i];
+        //	Prevent any non-image file type from being read.
```
Comment (squallstar) [inline js/build.js]: indentation looks weird

File: js/build.js [hunk 2]
```diff
@@ -351,6 +428,174 @@
...
+            document.body.focus();
+            switch (button) {
+              case 1:
+                cameraOptions.source = Camera.PictureSourceType.CAMERA;
+                return resolve(cameraOptions);
+              case 2:
+                cameraOptions.source = Camera.PictureSourceType.PHOTOLIBRARY;
+                return resolve(cameraOptions);
+              default:
+                return;
```
Comment (squallstar) [inline js/build.js]: `reject()` ?

File: js/build.js [hunk 3]
```diff
@@ -351,6 +428,174 @@
...
+      }
+
+      if (typeof navigator.camera === 'undefined') {
+        return Fliplet.Navigate.popup({
+          popupTitle: 'Sorry',
+          popupMessage: 'It looks like your app does not support file upload. Please contact the support team for more information.'
+        });
+      }
+
+      var popoverOptions = {
```
Comment (squallstar) [inline js/build.js:490]: indentation is all weird here again

File: js/build.js (hunk 2 above)
Comment (tonytlwu) [reply js/build.js]: This would've meant the user tapped on "Cancel", therefore it does nothing

**Review by squallstar — DISMISSED** [review]
please fix the indentation and code style (look at the changes here to see how weird it looks :))

File: js/build.js (hunk 2 above)
Comment (tonytlwu) [reply js/build.js]: Actually, I could add a `case 3:` for "Cancel" and add what you've suggested

File: js/build.js (hunk 3 above)
Comment (tonytlwu) [reply js/build.js:490]: I don't see anything weird with this one? 🤔

Comment (tonytlwu) [reply js/build.js:490]: I've trimmed all trailing whitespace and converted tabs to space. Text editor tells me there're no changes to save.

**Comment by tonytlwu** [comment]
@squallstar Pending your approval/response

**Pull request merged**